tensorflow/core/common_runtime/eager/eager_op_rewrite_registry_test.cc

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/core/common_runtime/eager/eager_op_rewrite_registry.h"
+#include "tensorflow/core/platform/test.h"
+
+namespace tensorflow {
+
+class TestEagerOpRewrite : public EagerOpRewrite {
+ public:
+  static int count_;
+  Status Run(EagerOperation* orig_op,
+             std::unique_ptr<tensorflow::EagerOperation>* out_op) override {
+    ++count_;
+    const tensorflow::AttrTypeMap* types;
+    bool is_function = false;
+    string kNewOp = "NoOp";
```

Comment:
nit: rename kNewOp -> new_op or something similar (since its not a const - alternatively make it const).

Reply:
Thanks. Fixed.